modules/react-native-wallet/ios/ApplePushProvisioning.m

```diff
@@ -0,0 +1,99 @@
+#import "ApplePushProvisioning.h"
+#import <React/RCTLog.h>
+#import <React/RCTUtils.h>
+#import <React/RCTBridge.h>
+#import <PassKit/PassKit.h>
+
+static NSString * const ModuleName = @"ApplePushProvisioning";
+static NSString * const ErrorPaymentPassUnsupported = @"payment_pass_unsupported";
+static NSString * const ErrorRootViewController = @"root_view_controller_error";
+static NSString * const ErrorCompletionHandler = @"completion_handler_error";
+
+// Token service providers. Add more if necessary
+static PKPaymentNetwork const CardNetwork = PKPaymentNetworkVisa;
```

Comment:
Please verify this line of code because it's causing the following error in xCode and blocking the iOS App from building:

`/ApplePushProvisioning.m:13:45 Initializer element is not a compile-time constant`